utils/scan_parameters.py

```diff
@@ -78,6 +78,14 @@ def PrepareYaml(name, proj):
     with open("%s.yml" % name, 'w') as file:
         yaml.dump(proj, file, default_flow_style=False)
 
+def ConfigureTestBench(mydir):
```

Comment:
The alternative to this solution is by editing the build.tcl, providing the path in the tb. OK, eventually the same I think.

Reply:
I see. I removed the tb data from the build file because it was then copying the whole file but perhaps there is a better trick such that this does not happen. If you tell me how you would modify the build file I can move to that solution.

Reply:
In my case, I added "add_files -tb ../../analysis-tools/data_to_txt_Norm" to the build.tcl, which will provide the relative path to the data file.

Reply:
Ok. Let me give a try and I'll add this fix. It's probably a bit cleaner than editing the test bench code.